src/server/queue_func.c

```diff
@@ -159,6 +160,15 @@ que_free(pbs_queue *pq)
 
 		pdef->at_free(pattr);
 	}
+	/* free default chunks set on queue */
+	pkvp = pq->qu_seldft;
+	if (pkvp) {
+		for (i=0; i < pq->qu_nseldft; ++i) {
```

Comment:
Add spaces around the '='

Reply:
I agree with you @bhroam, I think Hiren's concern about checking for null was that we don't dereference it when it is null.